modules/installation-mirror-repository.adoc

```diff
@@ -31,107 +35,66 @@ x509: certificate relies on legacy Common Name field, use SANs or temporarily en
 
 .Procedure
 
-Complete the following steps on the mirror host:
-
 . Review the
 link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
 to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
 
-. Set the required environment variables:
-.. Export the release version:
-+
-[source,terminal]
-----
-$ OCP_RELEASE=<release_version>
-----
-+
-For `<release_version>`, specify the tag that corresponds to the version of {product-title} to
-install, such as `4.5.4`.
-
-.. Export the local registry name and host port:
-+
-[source,terminal]
-----
-$ LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>'
-----
-+
-For `<local_registry_host_name>`, specify the registry domain name for your mirror
-repository, and for `<local_registry_host_port>`, specify the port that it
-serves content on.
-
-.. Export the local repository name:
-+
-[source,terminal]
-----
-$ LOCAL_REPOSITORY='<local_repository_name>'
-----
-+
-For `<local_repository_name>`, specify the name of the repository to create in your
-registry, such as `ocp4/openshift4`.
-
-.. Export the name of the repository to mirror:
-+
+. Set the required environment variables.
+.. Create the following `bash` script to set the variables, and name it `environment-variables.sh`.
```

Comment:
@dmesser, you mentioned that this sequence of commands looked far more involved than it actually should be. What do you think about this approach?

Reply:
Yeah, that makes a little bit easier 👍